stdlib/_operator.pyi

```diff
@@ -37,16 +37,16 @@ class _SupportsPos(Protocol[_T_co]):
 
 # Different to _typeshed.SupportsLessThan
 class _SupportsLT(Protocol):
-    def __lt__(self, other: Any) -> Any: ...
+    def __lt__(self, __other: Any) -> Any: ...
 
 class _SupportsGT(Protocol):
-    def __gt__(self, other: Any) -> Any: ...
+    def __gt__(self, __other: Any) -> Any: ...
 
 class _SupportsLE(Protocol):
-    def __le__(self, other: Any) -> Any: ...
+    def __le__(self, __other: Any) -> Any: ...
 
 class _SupportsGE(Protocol):
-    def __ge__(self, other: Any) -> Any: ...
+    def __ge__(self, __other: Any) -> Any: ...
 
```

Comment:
I regretted not adding an explanatory comment when I added these protocols (I don't think it's self-evident) -- while we're here, could you add them now?

```suggestion

# We get false-positive errors if e.g. `lt` does not have the same signature as `le`,
# so a broad union type is required for all four comparison methods
```

Reply:
Sure, done

Reply:
Thanks! 😀